src/Attestation/Attestation/Commands/GetAzureAttestation.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.Azure.Commands.Attestation.Models;
+using Microsoft.Azure.Commands.Attestation.Properties;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System;
+using System.Collections;
+using System.Management.Automation;
+using System.Runtime.InteropServices.ComTypes;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Rest.Azure;
+
+namespace Microsoft.Azure.Commands.Attestation
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Attestation", SupportsShouldProcess = true, DefaultParameterSetName = NameParameterSet)]
+    [OutputType(typeof(PSAttestation))]
+    public class GetAzureAttestation : AttestationManagementCmdletBase
+    {
+        #region Parameter Set Names
+
+        private const string NameParameterSet = "NameParameterSet";
+        private const string InputObjectParameterSet = "IdParameterSet";
+
+
+        #endregion
+
+        #region Input Parameter Definitions
+        /// <summary>
+        /// Attestation name
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            ParameterSetName = NameParameterSet,
+            HelpMessage = "Attestation Name.")]
+        [ResourceNameCompleter("Microsoft.Attestation", "ResourceGroupName")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Attestation object
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            ParameterSetName = InputObjectParameterSet,
+            ValueFromPipeline = true,
+            HelpMessage = "Attestation object to be queried.")]
+        [ValidateNotNullOrEmpty]
+        public PSAttestation InputObject { get; set; }
```

Comment:
@shleiAmy you can remove this parameter and the corresponding `InputObjectParameterSet` parameter set -- the object that the user provides to this parameter _should_ be the same thing that the cmdlet returns, so there isn't any value in having it

Reply:
removed InputObjectParameterSet parameters set